.eslintrc.yaml

```diff
@@ -18,7 +18,6 @@ globals:
 rules:
   no-unused-vars: ["error", { "argsIgnorePattern": "^_", "destructuredArrayIgnorePattern": "^_" }]
   react/jsx-filename-extension: [1, { "extensions": [".js", ".jsx"] }]
-  react/no-direct-mutation-state: off
```

Comment:
Could someone help me with a90938b7d10672c59e7b548e050c3d38dff5c42a? Currently, this is the only thing that is causing the CI job to [fail](https://github.com/nextstrain/auspice/actions/runs/4770552082/jobs/8481885730?pr=1665):

```
/home/runner/work/auspice/auspice/src/components/map/map.js
Error:   584:5  error  Do not mutate state directly. Use setState()  react/no-direct-mutation-state
```

The [`react/no-direct-mutation-state` doc page](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-direct-mutation-state.md) says that changes to `this.state` should only happen in the constructor, however I'm unsure how to refactor the code to do that.

Reply:
That's complaining about:

https://github.com/nextstrain/auspice/blob/1ff15addb24d7672c2d3802ecfef5a588b521714/src/components/map/map.js#L584

But is `currentBounds` actually _used_ anywhere? I don't see where it is.

It was introduced in 8a213ac8976c34df1182a441336bc18cc48401e2 and the last usage aside from the assignment above was seemingly removed in 43ee2ed6220944f57b5cc2f1eecaaa639f0e208b.

In general though, I think you could almost mechanically replace `this.state.X = Y` with approximately `this.setState({X: Y})`, as long as it was ok for subsequent code in the same tick to not notice the state change.

Reply:
Nice digging, that's helpful context! I've replaced a90938b7d10672c59e7b548e050c3d38dff5c42a with 172917602a382bc05ed62c3b247ccba38211af99...c7d3362439ee833661e2ec99af5bcb8c1105f451.